pages/common/mutt.md

```diff
@@ -0,0 +1,23 @@
+# mutt
+
+> Command-line email client.
+
+- Open the specified mailbox:
+
+`mutt -f {{mailbox}}`
+
+- Send an email and specify a subject and a cc recipient:
+
+`mutt -s {{subject}} -c {{cc@example.com}} {{recipient@example.com}}`
+
+- Send an email with files attached:
+
+`mutt -a {{file1}} {{file2}} -- {{recipient@example.com}}`
+
+- Specify a file to include as the message body:
+
+`mutt -i {{file}} {{recipient@example.com}}`
+
+- Specify a draft file containing the header and the body of the message:
+
+`mutt -H {{file}} {{recipient@example.com}}`
```

Comment:
@agnivade after some investigation, I found a reference that could be added here. According to https://en.wikipedia.org/wiki/Email#Header_fields, email headers must conform to IETF's [RFC 5322](https://tools.ietf.org/html/rfc5322). I think we could change the example description to:

`Specify a draft file containing the header and the body of the message, in RFC 5322 format:`